commentHandler.go

```diff
@@ -308,3 +366,18 @@ func checkTransition(requestedAction string, currentState string) (string, bool)
 
 	return "", false
 }
+
+//Function set milestones field to interface{} aka. null since library does not support that
```

Comment:
In go doc prefix every comment for a function header with the name of the function - i.e. `removeMilestone sets ... `